CellTypeScan.md

```diff
@@ -0,0 +1,87 @@
+# CellTypeScan Project
+
+### Introduction 
+Assigning types to cellular expression profiles remains a manual process much of the time, based on specialist knowledge of the expression of key genes in specific biological contexts. Large atlas efforts are ongoing but remain incomplete, meaning that the set of possible reference cells types is still poorly defined, and comprehensive controlled vocabularies for annotating cell types are not available. Thus, author-provided cell type information is likely to remain incomplete even when reference atlases and ontologies are complete.
+
+For these reasons, it is desirable to use automated methods to provide putative assignments of cells to type. Many cell type assignment methods exist, but their relative merits and effectiveness in real-world contexts are unknown. A key objective for the CellTypeScan project is to incorporate cell typing methods into the SCXA workflow, as outlined below. 
```

Comment:
You say CellTypeScan here as if you're introduced the concept previously. Could do with an early sentence somewhere along the lines of "CellTypeScan' is a project to componentise and integrate cell type assignment methods into a common framework for execution and evaluation" - or some other intro sentence of your own devising.

Reply:
You could even mention that it is inspired in InterProScan (hence the name), as an InterProScan for cell types.

Reply:
@pinin4fjords @pcm32 There has been no specific name for this project before, but we can stick with CellTypeScan. I'll add a couple of sentences to the introduction, also including InterProScan.